src/main/features/response/form/models/howMuchOwed.ts

```diff
@@ -0,0 +1,37 @@
+import { Serializable } from 'models/serializable'
+import { IsDefined, IsPositive, MaxLength } from 'class-validator'
+import { IsNotBlank } from 'app/forms/validation/validators/isBlank'
+import { ValidationErrors } from 'features/validationErrors'
+
+export class HowMuchOwed implements Serializable<HowMuchOwed> {
+
+  @IsDefined({ message: ValidationErrors.AMOUNT_REQUIRED })
+  @IsPositive({ message: ValidationErrors.VALID_OWED_AMOUNT_REQUIRED })
+  amount?: number
```

Comment:
Please add a newline between fields for readability.

Reply:
The validation on this field needs updating, please see `ClaimAmountRow#amount` for reference.